storage/store/test.go

```diff
@@ -46,10 +62,49 @@ func handleGetError(t *testing.T, err error, found bool) {
 	}
 }
 
+// NewTestConsulInstance allows to provide new Consul instance for tests
+// This is a private Consul server instantiation as done in github.com/ystia/yorc/v4/testutil
+// This allows avoiding cyclic dependencies with deployments store package
+func NewTestConsulInstance(t testing.TB) (*testutil.TestServer, *api.Client, config.Configuration) {
```

Comment:
Same as above looks like messy to define config within this function

Reply:
Fixed